src/asset-swapper/utils/market_operation_utils/path.ts

```diff
@@ -80,36 +80,35 @@ export class Path implements IPath {
         return [...this.ordersByType.nativeOrders, ...this.ordersByType.bridgeOrders, ...twoHopOrders];
     }
 
+    /**
+     * Returns `OptimizedOrdersByType` with slippage applied (Native orders do not have slippage).
+     * @param maxSlippage maximum slippage. It must be [0, 1].
+     * @returns orders by type by with slippage applied when applicable.
+     */
+    public getSlippedOrdersByType(maxSlippage: number): OptimizedOrdersByType {
+        checkSlippage(maxSlippage);
+
+        const { nativeOrders, twoHopOrders, bridgeOrders } = this.getOrdersByType();
```

Comment:
just something to think about -- is it possible we'll have different order types or orders with > 2 hops in the future? it might make sense to have a slippage applier class that loops through orders and applies slippage